src/irmin-pack/traverse_pack_file.ml

```diff
@@ -56,7 +61,65 @@ end = struct
      as requests for more data. *)
   exception Not_enough_buffer
 
-  type index_binding = { key : Hash.t; data : Index.value }
+  type index_value = int63 * int * Pack_value.Kind.t
+  [@@deriving irmin ~equal ~pp]
```

Comment:
Small preference for `value_t` to `Pack_index` instead.

Reply:
The typerepr of `value` isn't exposed in `Pack_value.S` (nor in `Index.S`). 

It should maybe be added to `Index.S`.

Reply:
Yeah, I was thinking of adding it to `Index.S` but my comment was poor :) Either way, no big deal.